admin/inc/inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+# *** LICENSE ***
+# This file is part of BlogoText.
+# http://lehollandaisvolant.net/blogotext/
+#
+# 2006      Frederic Nassar.
+# 2010-2016 Timo Van Neerden.
+#
+# BlogoText is free software.
+# You can redistribute it under the terms of the MIT / X11 Licence.
+#
+# *** LICENSE ***
+
+/**
+ * all file in /admin/inc/*.php
+ * must be include here
+ */
+
+
+require_once get_path('/inc/auth.php');
```

Comment:
Et cette erreur :

    var_dump(dirname(__file__, 2));
    Warning:  dirname() expects exactly 1 parameter, 2 given in /mnt/master/www/local/blogotext/admin/inc/inc.php on line 18